fearless/en.lproj/Localizable.strings

```diff
@@ -1249,3 +1249,4 @@ To find out more, contact %@";
 "vesting.locked.title" = "Vesting Locked";
 "vesting.claim.disclaimer.title" = "Important Notice Regarding Token Claims:";
 "vesting.claim.disclaimer.text" = "Due to the unique vesting schedules of each parachain, our app is unable to display the quantity of locked tokens eligible for claiming. Please be advised that initiating a claim may be impractical if the prospective amount is marginal and comparable to the transaction fee involved. For comprehensive insights into your locked balances, consult the Subscan blockexplorer. We urge you to assess the information carefully and proceed at your own discretion.";
+"common.and.others.placeholder" = "%s & others";
```

Comment:
please don't forget about adding string to localise and also export strings for all locale